src/sidebar/components/test/markdown-editor-test.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const { createElement, render } = require('preact');
+const { act } = require('preact/test-utils');
+const { mount } = require('enzyme');
+
+const { LinkType } = require('../../markdown-commands');
+const MarkdownEditor = require('../markdown-editor');
+
+describe('MarkdownEditor', () => {
+  const formatResult = {
+    text: 'formatted text',
+    selectionStart: 0,
+    selectionEnd: 0,
+  };
+  const fakeMarkdownCommands = {
+    convertSelectionToLink: sinon.stub().returns(formatResult),
+    toggleBlockStyle: sinon.stub().returns(formatResult),
+    toggleSpanStyle: sinon.stub().returns(formatResult),
+    LinkType,
```

Comment:
Curious why the others are mocked but this one is not?

Reply:
`LinkType` is an enum (that is, object mapping names to constants) whereas the others are functions.